packages/core/src/electron-main/electron-main-window-service-impl.ts

```diff
@@ -30,9 +30,11 @@ export class ElectronMainWindowServiceImpl implements ElectronMainWindowService
         if (external) {
             shell.openExternal(url);
         } else {
-            this.app.createWindow().then(electronWindow => {
-                electronWindow.loadURL(url);
-            });
+            this.app.getLastWindowOptions().then(options =>
```

Comment:
Last window options logic shouldn't be handled in this service. `openNewWindow` is generic and shouldn't care about previous windows?